schema/settings.schema.json

```
@@ -560,6 +560,14 @@
         "type": "string"
       },
       "default": {}
+    },
+    "orgs.allMembersBatchChangesAdmin": {
+      "description": "Enables/Disables org-level admin access for Batch Changes to all members of an org",
```

(Sorry I'm such a word-nit-smith 😅)

```suggestion
      "description": "If enabled, all members of the org will be treated as admins (e.g. can edit, apply, delete) for all batch changes created in that org.",
```